### DISCUSSION TITLE
deploy to aws ecs

### DISCUSSION THREAD

**Discussion by m4ush3r** [shlinkio/discussions#64] [Help wanted] [answered]
Hi, does anybody have deployed shlink into aws ecs?

I'm trying to deploy it but the container continue to reboot over and over again

Multiple Cloudwatch erros:

  `An exception occurred in the driver: SQLSTATE[HY000] [2002] Operation timed`  
  
 `In ExceptionConverter.php line 103:`

 `Creating fresh database if needed...`

**Comment by acelaya** [comment, marked as answer]
To me it looks like the EC instance where you are running Shlink does not have access to the database.

**Reply by m4ush3r** [reply]
thanks, you where right, something with the SG
